test/videojs-chapter-thumbnail-template.spec.js

```diff
@@ -14,6 +14,19 @@ describe('chapter-thumbnail-template.js', function () {
         );
     });
 
+    it('should return the modified default template when receiving non JSON data.', function () {
+        expect(chapterThumbnailTemplate({
+            text: "example"
+        })).toBe(
+            `
+<div class="vjs-chapters-thumbnails-item">
+    <img class="vjs-chapters-thumbnails-item-image" src="" />
```

Comment:
also, I might consider looking into not including the image if only a `title` was passed in?

you will have to double check, as I haven't looked at the code base yet.

thoughts?